### ISSUE TITLE
Socks5 Proxy Error

### ISSUE THREAD

**Issue by regalstreak** [zeal1209/XFakePlayers#2]
Hi zeal! Fan of your work.
Thank god you using github now!
Here is the problem.
Ive made a socks5 proxy server with my ubuntu vps using danted server
But when i connect, this is the log:

```15:07:31 - euWmXs (5.196.72.52:27015) disconnected, reason: "Cannot connect to 5.196.72.52:27015 via xxx.xxx.xxx.xxx:60256"```

Can connect perfectly with my browser. Please help @zeal1209 Thanks

**Comment by regalstreak** [comment]
Ok so no reply, but i found somewhat of a fix. Use proxycap and put the socks proxy in there. That is working good.

http://www.proxycap.com

Thanks!

**Comment by zeal1209** [comment]
Hello, regalstreak!

During the development of support socks5 proxy, I tested the program on public proxy servers found in Google, and the program was working perfectly. Make sure that your proxy server is stable, and can be seen in the global Internet network. Also, socks5 proxy server must support UDP connects and associations.

**Comment by regalstreak** [comment]
@zeal1209 yes ive checked that. Still it wont work. The proxy is udp stable as ive configured it that way and have checked it with other udp programs. Ok so i also tried some other proxies that were udp working. But still im getting the same problem :P can you please check again? Thanks brother!
PS: Please close this issue! My enemies would be hunting for me lol.

**Comment by regalstreak** [comment]
@zeal1209 new fix somewhere: https://www.neugomon.ru/threads/461/

**Comment by regalstreak** [comment]
works for hlds buffer v3 and for xfakeplayers

**Comment by poomato** [comment]
@regalstreak Could you tell us where you were able to find working proxies? 

I must have tested 100 socks5 proxies and always get the error "waiting for socks5 proxy association"

**Comment by regalstreak** [comment]
I couldnt use them with hlds buffer @poomato I had to tunnel my traffic with some program called proxycap. Ive given the link above. Proxies are found on google

**Comment by poomato** [comment]
@regalstreak Yes I've tried proxycap but none of the proxies I use ever work. 

Can you give me an example proxy that works for you, so I can see if it's a problem on my end?
